Classes/Model/Dto/Settings.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace StefanFroemken\ExtKickstarter\Model\Dto;
+
+use TYPO3\CMS\Core\Configuration\Exception\ExtensionConfigurationExtensionNotConfiguredException;
+use TYPO3\CMS\Core\Configuration\Exception\ExtensionConfigurationPathDoesNotExistException;
+use TYPO3\CMS\Core\Configuration\ExtensionConfiguration;
+
+class Settings
```

Comment:
What is the benefit of this class? It's nearly a copy of ExtConf. This DTO is not modifiable anymore.